stories/IssueArchives.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+    <div class="issue-archives">
+        <div class="issue-archives__tabs">
+            <ul>
+                <li v-for="tab in archives" :key="tab.year">
+                    <button
+                        @click="activetab = tab.year"
+                        :aria-selected="activetab === tab.year"
```

Comment:
There are a couple more things to do follow the inclusive components guidance on [tabbed interfaces](https://inclusive-components.design/tabbed-interfaces/). First, we need to add `role="tablist"`, `role="presentation"`, `role="tab" and `role="tabpanel"` as done in [this example](https://codepen.io/heydon/pen/veeaEa/). Second, we need to add keyboard commands to support tabbing past the tabs and using left/right arrows to navigate the tabs.

This will be one the later things we add so let's plan to pair program this. Maybe we can even have a general session if others want to learn about these techniques.